onadata/apps/restservice/migrations/0003_remove_deprecated_services.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('restservice', '0002_add_related_name_with_delete_on_cascade'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='restservice',
+            name='name',
+            field=models.CharField(max_length=50, choices=[('kpi_hook', 'KPI Hook POST')]),
```

Comment:
I've never been a fan of Django generating these migrations. Does any database actually enforce this? It's not like the migration is going to delete all the rows in the table whose `name` is not in the `choices`.

Let's go ahead and put it in, though, so that Django doesn't keep telling us that we need to generate new migrations.